CruCentralCoast/Extensions/UIViewControllerExtensions.swift

```diff
@@ -6,13 +6,15 @@
 //  Copyright © 2018 Landon Gerrits. All rights reserved.
 //
 
-import UIKit
+import WebKit
 
 public extension UIViewController {
     func insertProfileButtonInNavBar() {
         if let largeTitleView = self.navigationController?.navigationBar.subviews.first(where: {
             String(describing: type(of: $0)) == "_UINavigationBarLargeTitleView" } ) {
             
+            NotificationCenter.default.addObserver(self, selector: #selector(self.popToRootViewController), name: MainTabBarController.tabBarChangedNotification, object: nil)
```

Comment:
Is this method the right place for this?

Reply:
I feel like the best place would be in the viewDidLoad() method on each tab's root view controller

Reply:
Oh yeah duh that's smart. ill do that

Reply:
i think the original reason i did it here was to make it so i didn't need to put that line of code in each individual view controller.  however, i agree that it doesn't really seem to fit in this method. I went ahead and moved it.